config/jobs/ci-infra/ci-infra-presubmits.yaml

```diff
@@ -67,3 +67,15 @@ presubmits:
           requests:
             cpu: 6
             memory: 2Gi
+  - name: pull-ci-infra-check-renovate-config
+    cluster: gardener-prow-build
+    always_run: true
+    annotations:
+      description: Validates renovate config on pull requests to master branch
+    decorate: true
+    spec:
+      containers:
+      - name: renovate
+        image: ghcr.io/renovatebot/renovate:37.200.0
```

Comment:
You probably want to add a custom manager to keep this image up to date (and other images in job configs). You could use something like this:

```json5
    {
      // generic detection for pod-like image specifications
      "fileMatch": ["^config\/jobs\/.+\\.yaml$"],
      "matchStrings": ["image: (?<depName>.*?):(?<currentValue>.*?)\\s"],
      "datasourceTemplate": "docker"
    },
```